tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -713,6 +715,20 @@ EOF
 	info "Create /etc/resolv.conf file in rootfs if not exist"
 	touch "$dns_file"
 
+	if [ "${KATA_BUILD_CC}" == "yes" ]; then
+		info "Integrate pause image inside rootfs for CC"
+		pause_repo="$(get_package_version_from_kata_yaml externals.pause.repo)"
+		pause_version="$(get_package_version_from_kata_yaml externals.pause.version)"
+		[ -n "pause_repo" ] || die "failed to get pause image repo"
+		[ -n "pause_version" ] || die "failed to get pause image version"
+
+		skopeo copy "${pause_repo}":"${pause_version}" oci:pause:"${pause_version}"
+		umoci unpack --image pause:"${pause_version}"  "${ROOTFS_DIR}/pause_bundle"
+
+		info "Install init_trusted_storage script for CC"
+		install -o root -g root -m 0500 "${script_dir}/init_trusted_storage.sh" "${ROOTFS_DIR}/usr/bin/kata-init-trusted-storage"
```

Comment:
I think the trusted storage code should be removed from this commit as it's not related to pause image AFAIK and we have https://github.com/kata-containers/kata-containers/issues/8142 to re-introduce it and test it later

Reply:
Fixed.